projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot/sync-pilot.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="user$ | async as user">
+  <ng-container *ngIf="component$ | async as component">
+    <h3 *ngIf="!agent" class="table-header-title mb-0">
+      {{ 'fscommon.syncPilot.needHelp' | cxTranslate }}
+    </h3>
+    <div class="link" (click)="establishConnection(user, component, agent)">
+      <span *ngIf="agent; else noSpecificAgent" class="{{ agentIcon }}"></span>
+    </div>
+  </ng-container>
+</ng-container>
+
+<ng-template #noSpecificAgent>
+  <h4 class="{{ agentIcon }} table-header-value"></h4>
+  <p class="mb-0 text-truncate">
```

Comment:
This paragraph seems more logical when agent exists, so perhaps move it to the default case?

Reply:
Yes, but we are not tailoring this to be specific to any component since it's not generic then, right? We should perhaps implement one more input to handle this text?

Reply:
If we speak about functionality it is generic and the customer has two options to contact any available agent or to contact the specific agent. If we speak about design for me one we have on the comparison table is not generic:
![Screenshot 2021-11-25 at 10 40 15](https://user-images.githubusercontent.com/75082466/143417776-8333cf85-68ac-417c-a6db-81816ef54a61.png)
But in that case, the customer could extend our component and make his own design.

More generic is what I made to be default:

![Screenshot 2021-11-25 at 10 45 38](https://user-images.githubusercontent.com/75082466/143418152-f8514265-7540-4082-a982-9c6f662ae486.png)

For the comparison case then I need two inputs to handle the text.

I would rather leave any new idea for discussion and possible future improvement.